bw2data/project.py

```diff
@@ -43,14 +44,14 @@ def lockable():
     return False
 
 
-def new_uuid() -> str:
-    return uuid.uuid4().hex
+def new_snowflakeid() -> int:
```

Comment:
Sorry to be pedantic but I think we should either change the column type to `int` or have `new_snowflakeid` return a `str`… I am +0 on returning a `str` as it means I don’t have to rewrite the schema migration code.

Reply:
https://github.com/brightway-lca/brightway2-data/pull/200#discussion_r1822650914 :P

I've changed the `ProjectDataset` to use `IntegerField` for revisions.